subsys/caf/modules/Kconfig.buttons

```diff
@@ -22,7 +22,7 @@ config CAF_BUTTONS_PM_EVENTS
 	bool "Power management events support"
 	depends on CAF_PM_EVENTS
 	help
-	  React on power management events in buttons module.
+	  React on power management events in buttons module and submit wake up event.
```

Comment:
In this case this would sound better:
Support power management events.

Details are provided in the documentation. wut?

Reply:
changed